documentation/TechnicalDesignDocument.tex

```diff
@@ -191,11 +197,33 @@
     The first BoundingBox that is intersected by the ray will be the selected object.
     Casting the rays from the camera origin and locating the first intersected BoundingBox will prevent that 3D objects in the background will be selected.
 
+    \subsubsection{View and Game Communication}
+    View and editor (game) communication has everything to do with the way the two applications will work together.
+    The view is built in Java using libGDX, a graphics library and the editor is built using Scala.
```

Comment:
> The view is built in Java using libGDX, a graphics library and the editor is built using Scala.
The view is built in Java using libGDX, a graphics library, and the editor is built using Scala.

Reply:
Same thing here. Are you going to commit the changes or should I?